pkg/providers/cloudstack/testdata/cluster_main.yaml

```diff
@@ -74,8 +74,8 @@ spec:
     label: "data_disk"
```

Comment:
why are we messing with this file? It's for unit testing purposes, not documentation purposes